packages/near-fast-auth-signer/src/lib/controller.ts

```diff
@@ -269,15 +303,16 @@ class FastAuthController {
     return this.connection.provider.block({ finality: 'final' });
   }
 
-  async signAndSendActionsWithRecoveryKey({
+  async createSignedDelegateWithRecoveryKey({
     oidcToken,
     accountId,
     recoveryPK,
     actions,
   }) {
     const GET_SIGNATURE_SALT = CLAIM + 3;
     const GET_USER_SALT = CLAIM + 2;
-    const localKey = await this.getKey(`oidc_keypair_${oidcToken}`);
+    const localKey = await this.getKey(`oidc_keypair_${oidcToken}`) || await this.getLocalStoreKey(`oidc_keypair_${oidcToken}`);
```

Comment:
Duplicated line. If this will be a common scenario does it makes sense create a method for this get?

Reply:
Dependent on above discussion re multiple keystores so leaving as is for now